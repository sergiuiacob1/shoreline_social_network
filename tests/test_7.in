12
11
2 3
4 5
5 6
6 4
7 8
8 9
9 7
9 10
10 11
11 12
12 10
4
1 2
2 3
4 6
7 12