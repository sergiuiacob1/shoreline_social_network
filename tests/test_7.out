Distance between 1 and 2: -1
Distance between 2 and 3: 1
Distance between 4 and 6: 1
Distance between 7 and 12: 3
